Cargo.toml

```diff
@@ -5,21 +5,27 @@ description = "Implementation the car files for iroh"
 version = "0.2.1"
 edition = "2021"
 license = "Apache-2.0/MIT"
-rust-version = "1.65"
+rust-version = "1.60.0"
```

Comment:
adjusted to match the existing subconscious repo. I don't have a strong opinion on the version

Reply:
If it works, I am fine with it. But this also needs updating in the ci files to make sure the checks on CI are correct